source/docs/software/telemetry/robot-telemetry-with-annotations.rst

```diff
@@ -0,0 +1,295 @@
+Robot Telemetry with Annotations (Java Only)
+============================================
+
+At its most basic, telemetry with annotations can be accomplished with only a few lines of code:
```

Comment:
Some type of lead-in about this being new for 2025 WPIlib I think would be helpful

Reply:
I agree. Is there a standard format we should follow (such as prepending "New for 2025" to the title)?

Reply:
Not really, but here's one example. https://docs.wpilib.org/en/stable/docs/software/networktables/nt4-migration-guide.html#migrating-from-networktables-3-0-to-networktables-4-0